modules/openapi-generator/src/main/resources/ruby-client/partial_model_generic.mustache

```diff
@@ -51,6 +51,17 @@
       }
     end
 
+    # List of attributes with nullable: true
+    def self.openapi_nullable
+      [
+        {{#vars}}
+        {{#isNullable}}
+          :'{{{name}}}'{{#hasMore}},{{/hasMore}}
```

Comment:
indent seems to be too depth.

Reply:
And what about defining it as a hash instead similar to self.openapi_types?

If an object contains a lot of nullable properties, using `include?` may result in performance issues.

Reply:
I don't see usage of key-value, but what about using `Set` instead of `Array`, it should have better performance for include?

Reply:
I don't like the format 
```
def self.openapi_nullable
      Set.new([
      ])
end
```
is it possible to define it in template in a way that empty set won't generate new line?

Reply:
Hmm... it seems to be difficult because there is no way to directly enumerate nullable attributes in template.
@wing328 Do you have any ideas?

By the way, I don't like this format, but it's not acceptable (Rubocop also allows this), so I think this PR can be merged.
Isn't it okay to make another PR for reformatting?

Reply:
Set is definitely better than array in this case.

About empty set, I think it's acceptable for day 1 requirement. We can always improve it later with another PR.